src/components/pages/document.yml

```diff
@@ -68,3 +68,16 @@ document:
   submitInput:
     en: Submit input
     de: Eingabe senden
+  proposedSectionsAlertMarkdown:
+    en: |
+      Apply or discard the proposed sections one by one or click "Apply all".
+      \
+      Unapplied sections are only visible until you leave the current view.
+    de: |
+      Übernimm oder verwirf die vorgeschlagenen Abschnitte.
```

Comment:
Please help with the German equivalent of this first sentence 😬

Reply:
Übernimm oder verwirf die vorgeschlagenen Abschnitte jeweils individuell oder klicke auf "Alle übernehmen"

*(Given we change also "Alle anwenden" to "Alle übernehmen")*